README.md

```diff
@@ -40,8 +40,8 @@ A few Habitat packages must be built before the infrastructure can be provisione
 - **infra-linux-hardening** is used to harden the servers and is loaded as part of the [base applications](habitat-plans/infra-linux-base-applications-with-hardening/Policyfile.rb)
 - **infra-linux-base-applications** is used to load base applications, including **inspec-linux-audit**
 - **consul** is used as a backend to Vault and is used in [infrastructure-terraform/vault](infrastructure-terraform/vault)
-- **vault** used in [infrastructure-terraform/vault](infrastructure-terraform/vault)
-- **jenkins** used in used in [infrastructure-terraform/jenkins](infrastructure-terraform/jenkins)
+- **vault** is used in [infrastructure-terraform/vault](infrastructure-terraform/vault)
```

Comment:
Fixed some typos here but this structure needs to be more specific about the two directions you can go:

If you choose not to build these packages under your own origin, you may not be able to do the 4th example (as you need to be able to push changes to these packages). Additionally, you may only need to build some of these packages (i.e. no need to build Jenkins under your own origin).

Reply:
Resolved in #2